web/modeltr/document.py

```diff
@@ -35,6 +36,9 @@ def __init__(self, **kwargs):
                 # store types of each document entry
                 types.update({item[0]: item[1]._type})
 
+                if item[1].__name__ == 'trSaveTimestamp':
```

Comment:
For better clarity, I would suggest to create `member_name = item[0]` and `member_value = item[1]` and work with them afterwards.

Working with item[0] and item[1] works, but I needed to look it up in the docs to be sure what is being treated here.

Reply:
I am not entirely sure, why `trSaveTimestamp` has the special handling and others not.

Why is document_updated_property save in this case and in others it is not?

Reply:
If you want to update just timestamp, maybe better to name the variable in a way to reflect the timestamp relation somehow.

Reply:
`trSaveTimestamp` is meant to be datetype especially for storing the datetime when the doc is saved.

Reply:
I understand, but that was not my point - I am not sure why the variable `document_updated_property` is set only in case we manipulate trSaveTimestamp.

So maybe something like `updated_time_stamp_property`? Not sure I follow.

Also, please add:
```
member_name = item[0]
member_value = item[1]
```
...and please replace references to item[x] with these more explicative names. Thank you